From: KevinDuan <duankaiwen@kylinos.cn>
Date: Fri, 14 Aug 2026 08:12:11 +0000
Subject: Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translation: openkylin-huanghe/peony-menu-plugin-mate-terminal
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/peony-menu-plugin-mate-terminal/it/
---
 .../translations/peony-mate-terminal-extension_it.ts           | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/peony-menu-plugin-mate-terminal/translations/peony-mate-terminal-extension_it.ts b/peony-menu-plugin-mate-terminal/translations/peony-mate-terminal-extension_it.ts
index 7cfed7e..95d681d 100644
--- a/peony-menu-plugin-mate-terminal/translations/peony-mate-terminal-extension_it.ts
+++ b/peony-menu-plugin-mate-terminal/translations/peony-mate-terminal-extension_it.ts
@@ -7,17 +7,17 @@
         <location filename="../mate-terminal-menu-plugin.cpp" line="240"/>
         <location filename="../mate-terminal-menu-plugin.cpp" line="259"/>
         <source>Open Directory in Terminal</source>
-        <translation type="unfinished"></translation>
+        <translation>Apri terminale</translation>
     </message>
     <message>
         <location filename="../mate-terminal-menu-plugin.h" line="43"/>
         <source>Peony-Qt Mate Terminal Menu Extension</source>
-        <translation type="unfinished"></translation>
+        <translation>Estensione terminale di UKUI File Manager</translation>
     </message>
     <message>
         <location filename="../mate-terminal-menu-plugin.h" line="44"/>
         <source>Open Terminal with menu</source>
-        <translation type="unfinished"></translation>
+        <translation>Usa il menu contestuale per aprire il terminale</translation>
     </message>
     <message>
         <source>Open Terminal with menu.</source>
@@ -33,12 +33,12 @@
     <message>
         <location filename="../mate-terminal-menu-plugin.cpp" line="170"/>
         <source>Open terminal fail</source>
-        <translation type="unfinished"></translation>
+        <translation>Apertura del terminale non riuscita</translation>
     </message>
     <message>
         <location filename="../mate-terminal-menu-plugin.cpp" line="171"/>
         <source>Open terminal failed, did you removed the default terminal?  If it&apos;s true please reinstall it.</source>
-        <translation type="unfinished"></translation>
+        <translation>Apertura del terminale non riuscita, hai disinstallato il terminale predefinito? In tal caso, reinstallalo.</translation>
     </message>
     <message>
         <source>Open terminal failed, did you removed the default terminal?  If you have reinstalled it please logout and relogin.</source>
